sensor/AdblockPlugin.js

```diff
@@ -269,17 +292,21 @@ class AdblockPlugin extends Sensor {
       });
     }
 
-    async cleanUpFilter() {
-      const config = await this.getAdblockConfig();
-      for (const key in config) {
-        const file = `${dnsmasqConfigFolder}/${key}_adblock.conf`;
-        try {
-          if (fs.existsSync(file)) {
-            await fs.unlinkAsync(file);
+    _cleanUpFilter(config) {
+      try {
+        const result = []
+        if (typeof config == 'object') {
+          for (const key in config) {
+            if(config[key] == "on") result.push(key+"_adblock.conf")
           }
-        } catch (err) {
-          log.error(`Failed to delete file: '${file}'`, err);
         }
+        fs.readdirSync(dnsmasqConfigFolder).forEach(file => {
+          if (file.endsWith('_adblock.conf') && !result.includes(file)) {
```

Comment:
There are other group-level adblock config files ending with _adblock.conf. Need to use another way to pick the adblock domain list file. E.g., use a different suffix for domain list file.